src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -326,6 +360,79 @@
                     "description": "Recordings of test runs from a single node"
                 }
             }
+        },
+        "raft_state": {
+            "id": "raft_state",
+            "description": "Raft state",
+            "properties": {
+                "leader_info": {
+                    "type": "raft_leader_state",
+                    "description": "Leader info"
+                },
+                "followers": {
+                    "type": "array",
+                    "items": {
+                        "type": "raft_follower_state"
+                    },
+                    "description": "Raft followers info"
+                }
+            }
+        },
+        "raft_leader_state": {
+            "id": "raft_leader_state",
+            "description": "Raft leader state",
+            "properties": {
+                "id": {
+                    "type": "int",
+                    "description": "Node id"
+                },
+                "committed_log_index": {
+                    "type": "long",
+                    "description": "Commited log index"
+                },
+                "dirty_log_index": {
+                    "type": "long",
+                    "description": "Dirty log index"
+                }
+            }
+        },
+        "raft_follower_state": {
+            "id": "raft_follower_state",
+            "description": "Raft follower state",
+            "properties": {
+                "id": {
+                    "type": "int",
+                    "description": "Node id"
+                },
+                "is_learner": {
+                    "type": "boolean",
+                    "description": "Is node role learner"
+                },
+                "committed_log_index": {
+                    "type": "long",
+                    "description": "Commited log index"
+                },
+                "dirty_log_index": {
+                    "type": "long",
+                    "description": "Dirty log index"
+                },
+                "match_index": {
+                    "type": "long",
+                    "description": "Match index"
+                },
+                "ms_since_last_heartbeat": {
+                    "type": "long",
+                    "description": "Ms since last heartbeat"
+                },
+                "is_live": {
+                    "type": "boolean",
+                    "description": "Is node live"
+                },
+                "under_replicated": {
+                    "type": "boolean",
+                    "description": "Is node under replicated"
+                }
+            }
```

Comment:
Can we return content from `follower_index_metadata` instead of the follower metrics. This is inteded to be a debug API it is better to expose to much than not enough